docs/src/_includes/components/docs-index.html

```diff
@@ -10,16 +10,16 @@
 </li>
 {%- endmacro %}
 
-<nav class="docs-index" id="docs-index">
-    <button class="docs-index-toggle" aria-label="Index" id="js-docs-index-toggle" hidden>
```

Comment:
This is initially hidden because users without JavaScript shouldn't see this. For those with JavaScript, a script will unhide this.